src/WebView.windows.tsx

```diff
@@ -118,19 +123,22 @@ const WebViewComponent = forwardRef<{}, WindowsWebViewProps>(({
   const webViewContainerStyle = [styles.container, containerStyle];
 
   const NativeWebView
-  = useWebView2? RCTWebView2 : RCTWebView;
+  = useWebView2 ? RCTWebView2 : RCTWebView;
```

Comment:
nit: extra space

Reply:
Felt like it was neater to have a space between all the variables in the ternary